.github/workflows/cross-compiles.yml

```diff
@@ -0,0 +1,33 @@
+name: Cross Compile
+
+on: [pull_request, push]
+
+jobs:
+  cross-compilation:
+    strategy:
+      fail-fast: false
+      matrix:
+        # The platform matrix strings are two fields separated with a colon.
+        # The first field is the package to install using apt.
+        # The second field is passed to Configure as command line arguments.
+        platform: [
+          'gcc-aarch64-linux-gnu:-cross-compile-prefix=aarch64-linux-gnu- linux-aarch64',
+          'gcc-alpha-linux-gnu:--cross-compile-prefix=alpha-linux-gnu- linux-alpha-gcc',
+          'gcc-arm-linux-gnueabi:--cross-compile-prefix=arm-linux-gnueabi- linux-armv4',
+          'gcc-arm-linux-gnueabihf:--cross-compile-prefix=arm-linux-gnueabihf- linux-armv4',
+          'gcc-mips-linux-gnu:--cross-compile-prefix=mips-linux-gnu- linux-mips32',
+          'gcc-mipsel-linux-gnu:--cross-compile-prefix=mipsel-linux-gnu- linux-mips32',
+          'gcc-powerpc64le-linux-gnu:--cross-compile-prefix=powerpc64le-linux-gnu- linux-ppc64le',
+          'gcc-riscv64-linux-gnu:--cross-compile-prefix=riscv64-linux-gnu- linux64-riscv64',
+          'gcc-s390x-linux-gnu:--cross-compile-prefix=s390x-linux-gnu- linux64-s390x',
+          'gcc-sparc64-linux-gnu:--cross-compile-prefix=sparc64-linux-gnu- linux64-sparcv9',
+        ]
```

Comment:
I believe that if you make each line dictionaries, you won't have to play with `cut`:

``` yaml
      platform: [
        { target: gcc-aarch64-linux-gnu, config: '--cross-compile-prefix=aarch64-linux-gnu- linux-aarch64' },
        ...
```

Access with `${{ matrix.platform.target }}` and `${{ matrix.platform.config }}`

Might be worth trying out at least?

Reply:
Definitely.  Update pushed....
Much nicer.